connect/runtime/src/test/java/org/apache/kafka/connect/runtime/distributed/DistributedHerderTest.java

```diff
@@ -935,9 +936,6 @@ public Object answer() throws Throwable {
         herder.tick();
         assertTrue(connectorConfigCb.isDone());
         assertEquals(CONN1_CONFIG_UPDATED, connectorConfigCb.get());
-        // The config passed to Worker should
```

Comment:
In most cases, we only care about the connector name and connector class. The topics only matters when computing the tasks. With this said, it seems weird to use source connector config or sink connector config in those cases as we don't really care about whether the connector is source or sink. 

The source or sink only matters when computing tasks, may be we can use source connector config and sink connector config in those cases?